src/main/java/org/sciborgs1155/robot/Drive/DriveConstants.java

```diff
@@ -1,5 +1,22 @@
 package org.sciborgs1155.robot.Drive;
 
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.DifferentialDrivetrainSim;
+
 public class DriveConstants {
-  double maxSpeed = 5.0;
+  static double MAXSPEED = 5.0;
+  static Pose2d STARTINGPOSE = new Pose2d(5, 5, Rotation2d.fromRadians(0));
+  static DifferentialDrivetrainSim STARTINGDIFFERENTIALDRIVESIM =
```

Comment:
the inclination to put this here is totally reasonable. This, however, is not a constant. The drive sim should be in SimDrive, the constants that you give it (9, 8.5, etc.) should be here. If that's really annoying for you though (and i suspect it is) you can just put it in DriveSim and forget about moving the numbers to constants

Reply:
aye aye captain